pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -92,6 +94,10 @@ func getNodeThresholds(
 		if len(node.Status.Allocatable) > 0 {
 			nodeCapacity = node.Status.Allocatable
 		}
+		if len(resourceNames) == 1 && resourceNames[0] == ResourceMetrics {
+			// Make ResourceMetrics 100% => 100 points
+			nodeCapacity[ResourceMetrics] = *resource.NewQuantity(int64(100), resource.DecimalSI)
```

Comment:
nit: might be safer to double fail here. It expects that the plugin above does the validation correctly.